test/uk/gov/hmrc/tradergoodsprofilesdatastore/services/StoreRecordsServiceSpec.scala

```diff
@@ -18,202 +18,178 @@ package uk.gov.hmrc.tradergoodsprofilesdatastore.services
         val requestEori     = "GB123456789099"
 
-        when(mockRecordsRepository.getCountWithInactive(any())) thenReturn Future.successful(60000)
-
+        when(mockRecordsRepository.getCountWithInactive(any())) thenReturn Future.successful(totalRecordsNum)
+        when(mockRecordsSummaryRepository.set(any(), any())) thenReturn Future.successful(true)
         when(mockRecordsRepository.saveRecords(any(), any())) thenReturn Future.successful(true)
         when(mockRouterConnector.getRecords(any(), any(), any(), any())(any())) thenReturn (
           Future.successful(
             GetRecordsResponse(
               goodsItemRecords = getTestRecords(requestEori, recursivePageSize),
-              Pagination(totalRecordsNum, 1, 6, Some(2), None)
+              Pagination(totalRecordsNum, 0, 4, Some(1), None)
             )
           ),
           Future.successful(
             GetRecordsResponse(
               goodsItemRecords = Seq(getGoodsItemRecord(requestEori)),
-              Pagination(totalRecordsNum, 2, 6, Some(3), Some(1))
+              Pagination(totalRecordsNum, 1, 4, Some(2), Some(0))
             )
           ),
           Future.successful(
             GetRecordsResponse(
               goodsItemRecords = Seq(getGoodsItemRecord(requestEori)),
-              Pagination(totalRecordsNum, 3, 6, Some(4), Some(2))
-            )
-          ),
-          Future.successful(
-            GetRecordsResponse(
-              goodsItemRecords = Seq(getGoodsItemRecord(requestEori)),
-              Pagination(totalRecordsNum, 4, 6, Some(5), Some(3))
-            )
-          ),
-          Future.successful(
-            GetRecordsResponse(
-              goodsItemRecords = Seq(getGoodsItemRecord(requestEori)),
-              Pagination(totalRecordsNum, 5, 6, Some(6), Some(4))
+              Pagination(totalRecordsNum, 2, 4, Some(3), Some(1))
             )
           ),
           Future.successful(
             GetRecordsResponse(
               goodsItemRecords = Seq(getGoodsItemRecord(requestEori)),
-              Pagination(totalRecordsNum, 6, 6, None, Some(5))
+              Pagination(totalRecordsNum, 3, 4, None, Some(2))
             )
           ),
           Future.successful(
             GetRecordsResponse(
               goodsItemRecords = Seq(getGoodsItemRecord(requestEori)),
-              Pagination(totalRecordsNum, 1, totalRecordsNum, Some(2), None)
+              Pagination(totalRecordsNum, 1, totalRecordsNum, Some(1), None)
             )
           )
         )
-
-        val result = await(service.storeRecords(requestEori, None)(FakeRequest(), hc))
-
-        result.value shouldBe Done
-        verify(mockRouterConnector, times(7)).getRecords(any(), any(), any(), any())(any())
-        verify(mockRecordsRepository, times(6)).saveRecords(any(), any())
-        verify(mockRecordsRepository).getCountWithInactive(any())
+        val result = await(service.storeRecords(requestEori, None)(hc))
```

Comment:
Should be able to do
```scala
result shouldBe false
```
here instead of in the `eventually` as this indicates that it's separate from the things that are happening concurrently

Reply:
if i don't do it in the eventually it doesn't complete and errors - i think i need a success or assertion at the end of the eventually block to make it complete, this is why i put the result should be false there